src/doxygen.cpp

```diff
@@ -145,6 +145,7 @@ FileNameLinkedMap    *Doxygen::mscFileNameLinkedMap = 0;     // msc files
 FileNameLinkedMap    *Doxygen::diaFileNameLinkedMap = 0;     // dia files
 StringUnorderedMap    Doxygen::namespaceAliasMap;            // all namespace aliases
 StringMap             Doxygen::tagDestinationMap;            // all tag locations
+std::vector<QCString> Doxygen::tagFileVector;                // all tag file names
```

Comment:
a `std::set` is more efficient than a `std::vector` for checking if something is already processed (O(log(N)) vs O(N)).

Reply:
I don't think this will be noticeable as typical the number of tag files will be small (with CGAL I saw one subproject with 113 tag files and all other had less than 20). Also the set of tag files is just read once.

Adjusted the code from `std::vector` to `std::set`